gender_analysis/analysis/gender_adjective.py

```diff
@@ -367,3 +367,13 @@ def get_top_adj(full_results, num):
         top[gender] = sorted(excluded_results[gender].items(), reverse=True, key=lambda x:x[1])[0:num]
 
     return top
+
+def display_gender_adjectives(result_dict,num_to_return):
+    '''takes the results of find_gender_adj and prints out to the user the top number_to_return
```

Comment:
Instead of `'''` quotes for docstrings, could you use `"""` and separate them on a newline? It's more in line with the style of the other docstrings, and makes things a little cleaner in the long run.